src/runtime/manifest-parser.pegjs

```diff
@@ -1581,6 +1584,84 @@ SchemaPrimitiveType
     });
   }
 
+Adapter "an adapter, (e.g. adapter Foo(param: Person { name: Text }) => Friend { nickName: param.name } )"
+  = 'adapter' whiteSpace adapterName:AdapterName whiteSpace? '(' multiLineSpace? params:AdapterParamsDeclaration multiLineSpace? ')' whiteSpace? '=>' multiLineSpace? body:AdapterBodyDefinition eolWhiteSpace
```

Comment:
Should we be opinionated on whether `(` has to be adjacent to the name? e.g. `ToFriend(...` vs `ToFriend (...`

I would err on the side of no whitespace, wdyt?

Reply:
SGTM, I'll drop the whitespace. Seeing as how well opinionated formatting works for Go, I'm ok with not allowing arbitrary whitespace everywhere.